src/chatd.cpp

```diff
@@ -1705,6 +1755,8 @@ Chat::Chat(Connection& conn, Id chatid, Listener* listener,
     mLastReceivedId = info.lastRecvId;
     mLastSeenIdx = mDbInterface->getIdxOfMsgidFromHistory(mLastSeenId);
     mLastReceivedIdx = mDbInterface->getIdxOfMsgidFromHistory(mLastReceivedId);
+    std::string rsn = (!previewMode()) ? mDbInterface->getReactionSn() : std::string();
```

Comment:
No need to check the preview mode, but to save the chat in cache at the GroupChatRoom::ctor earlier than this method is called.
I'll fix it.